packages/flutter/lib/src/material/dialog.dart

```diff
@@ -176,6 +177,43 @@ class AlertDialog extends StatelessWidget {
   }
 }
 
+/// A button used in a [SimpleDialog].
+///
+/// See also:
+///
+///  * [SimpleDialog], for a dialog in which to use this button.
+///  * [showDialog], which actually displays the dialog and returns its result.
+///  * <https://material.google.com/components/dialogs.html#dialogs-simple-dialogs>
```

Comment:
also point to RaisedButton, FlatButton, etc, explaining when you'd prefer those.